openmetadata-ui/src/main/resources/ui/src/pages/ContainerPage/ContainerPage.tsx

```diff
@@ -661,17 +661,19 @@ const ContainerPage = () => {
   }, [containerName]);
 
   useEffect(() => {
-    if (tab === EntityTabs.CHILDREN) {
+    if (tab === EntityTabs.CHILDREN && hasViewPermission) {
       fetchContainerChildren(containerName);
     }
   }, [tab, containerName]);
 
   useEffect(() => {
-    getEntityFeedCount();
+    if (hasViewPermission) {
+      getEntityFeedCount();
+    }
   }, [containerName]);
```

Comment:
permission dependency?

Reply:
If user doesn't have view permission then, it will be unnecessary to fetch data which we will not be using!